data/json/npcs/TALK_TEST.json

```diff
@@ -319,6 +319,11 @@
       }
     ]
   },
+  {
+    "type": "talk_topic",
+    "id": "TALK_MORALE",
+    "responses": [ { "text": "This is a test for adding morale", "topic": "TALK_DONE", "effect": [ "morale_chat" ] } ]
```

Comment:
I'm fairly sure you can just do "effect": "morale_chat" if there's only a single effect.

But more to the point, please don't add talk_topics to TALK_TEST.json unless you're also adding test code to tests/npc_talk_test.cpp.

Reply:
Removed from talk_test and added the effect type to docs/NPC